keps/sig-network/2200-externalips-admission/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Block ExternalIPs via Admission Control
+kep-number: 2200
+authors:
+  - "@thockin"
+owning-sig: sig-network
+participating-sigs:
+  - sig-auth
+  - sig-security
+  - sig-api-machinery
+status: implementable
+creation-date: 2020-12-07
+reviewers:
+  - "@IanColdwater"
+  - "@tabbysable"
+approvers:
+  - "@tallclair"
+  - "@lavalamp"
+prr-approvers:
+  - "@johnbelamaric"
+see-also:
+  - "https://github.com/kubernetes/kubernetes/issues/97110"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
Hi there, 1.21 Enhancements Lead here. 
Based on the reviews, looks like this was changed to target stable for 1.21. Can we make sure this gets updated as well then?

Reply:
I discussed this at the PRR meeting today, we're OK with going straight to stable.